### DISCUSSION TITLE
images with link captions

### DISCUSSION THREAD

**Discussion by TWIISTED-STUDIOS** [aahnik/tgcf#368] [Q&A]
is it possible to set the filters of the bot to not forward images that have a link as a caption, but still allow other images to be sent.

**Comment by tissole** [comment]
Yes, it can be done. Go to Plugins, Filters, check the box `use this plugin`, click on `text` setting and put the link in the blacklist if is the same link on all photos. If are different links you have to check `regex` and search on the net for a regex expression that remove all the links.

This will block all files that contain a link. I don't know if is it possible to block only photos and let pass other files with links.

**Reply by TWIISTED-STUDIOS** [reply]
I tried this, I made sure that the regex worked by allowing just text and it blocked that so I made sure to enable no attachments as I don't want just messages, I then tried an image with the exact same links as the caption and it allowed them through, as this was my first thought also.

**Reply by tissole** [reply]
Please provide the regex rule and some links to try myself and see if I can replicate this issue.

**Reply by TWIISTED-STUDIOS** [reply]
Thank you very much I will get that to you as soon as I'm able to

**Reply by TWIISTED-STUDIOS** [reply]
this was the regex i was using, I have a feeling it is formated incorrectly (https|http)(://)\S+
